src/main/java/nextstep/optional/User.java

```diff
@@ -33,7 +36,10 @@ public static boolean ageIsInRange1(User user) {
     }
 
     public static boolean ageIsInRange2(User user) {
-        return false;
+        return Optional.ofNullable(user)
+            .map(User::getAge)
+            .filter(age -> 30 <= age && age <= 45)
```

Comment:
```suggestion
            .filter(age -> 30 <= age)
            .filter(age -> age <= 45)
```
이렇게 filter를 한 번 더 사용해주면 어떨까요?

Reply:
넵, 가독성 측면에서 좋네요. 그리고 filter를 여러번 쓸 수도 있군요.
감사합니다.